From 826e9b6608c71d167a3fb6fb5042eabc9df6b443 Mon Sep 17 00:00:00 2001
Message-Id: <826e9b6608c71d167a3fb6fb5042eabc9df6b443.1425610844.git.chang-joon.lee@intel.com>
In-Reply-To: <6fa2ab23adf6d7de781e906d04a0224410262d9b.1425610844.git.chang-joon.lee@intel.com>
References: <6fa2ab23adf6d7de781e906d04a0224410262d9b.1425610844.git.chang-joon.lee@intel.com>
From: Gaurav K Singh <gaurav.k.singh@intel.com>
Date: Fri, 13 Feb 2015 13:05:10 +0530
Subject: [PATCH 21/27] MUST_REBASE [VPG]: video/adf/intel: Initialize regs
 for DSI dual link

DSI dual link requires programming regs of both ports A & C. This patch fixes
the issue of having it only for one port that is either Port A or Port C.

MUST_REBASE: Since Google ADF framework is not in upstream and also
we cannot have two display drivers, other one being i915, we need to
work on i915 and ADF convergence path before it can be upstreamed.

Issue: GMINL-6006
Change-Id: If9ee1daf5aae23101896301f0432976cb8c4487b
Signed-off-by: Gaurav K Singh <gaurav.k.singh@intel.com>
---
 drivers/video/adf/intel/core/vlv/vlv_dc_config.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/adf/intel/core/vlv/vlv_dc_config.c b/drivers/video/adf/intel/core/vlv/vlv_dc_config.c
index 7a0a432..07c4791 100644
--- a/drivers/video/adf/intel/core/vlv/vlv_dc_config.c
+++ b/drivers/video/adf/intel/core/vlv/vlv_dc_config.c
@@ -317,7 +317,7 @@ static int vlv_initialize_port(struct vlv_dc_config *vlv_config,
 		dsi_ctx = &dsi_pipe->config.ctx;
 		for_each_dsi_port(port_no, dsi_ctx->ports) {
 			dsi_port = &disp->port.dsi_port[port_no];
-			vlv_dsi_port_init(dsi_port, port, pipe);
+			vlv_dsi_port_init(dsi_port, port_no, pipe);
 		}
 		break;
 	case INTEL_PIPE_DP:
-- 
1.7.9.5

